EnglishFilter/sections/adservers.txt

```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||a.teenspirithentai.com^$third-party
```

Comment:
to adservers_firstparty please `||a.teenspirithentai.com^`

Reply:
I agree with this.